ckanext/wet_boew/templates/wet_boew/macros/form.html

```diff
@@ -419,7 +419,7 @@
        data-module-field_url="{{ field_url }}" data-module-field_upload="{{ field_upload }}" data-module-field_clear="{{ field_clear }}" data-module-upload_label="{{ upload_label }}">
   {% endif %}
 
-  {{ input(field_url, label=url_label, id='field-image-url', placeholder=placeholder, value=data.get(field_url), error=errors.get(field_url), classes=['control-full']) }}
+  {{ input(field_url, label=url_label, is_required=is_required, id='field-image-url', placeholder=placeholder, value=data.get(field_url), error=errors.get(field_url), classes=['control-full']) }}
```

Comment:
will this put the "required" text just on the url box, not wrapping around both the url and upload fields? If so that would be confusing